sdks/python/apache_beam/utils/annotations.py

```diff
@@ -192,10 +190,4 @@ def inner(*args, **kwargs):
 # 'current' will be optional in both deprecated and experimental
 # while 'since' will be mandatory for deprecated.
 deprecated = partial(
-    annotate, label='deprecated', current=None, extra_message=None)
-experimental = partial(
-    annotate,
-    label='experimental',
-    current=None,
-    since=None,
-    extra_message=None)
+    annotate, label='deprecated', current=None, extra_message=None)
```

Comment:
do we not care about newlines at end of file?

Reply:
I just ran `yapf` over the whole codebase according to https://cwiki.apache.org/confluence/display/BEAM/Python+Tips#PythonTips-Runningyapfformattermanually

Reply:
makes sense, then isn't part of the config -- definitely don't want to be nitpicky [ manually ] about such things.